packages/generator-react-native-vector-icons/src/app/templates/font.podspec

```diff
@@ -0,0 +1,81 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = 'react-native-vector-icons-<%= packageName %>'
+  s.version      = package['version']
+  s.summary      = package['description']
+  s.homepage     = package['homepage']
+  s.license      = package['license']
+  s.authors      = package['author']
+
+  s.platforms    = { ios: min_ios_version_supported, tvos: '9.0', visionos: '1.0' }
+  s.source       = { git: package['repository']['url'], tag: "v#{s.version}" }
+
+  s.resources = 'fonts/*.ttf'
+<% if (copyCustomFonts) { -%>
+
+  s.script_phase = {
```

Comment:
this part seems to be at odds with how the podspec files in the individual packages look..?

Reply:
This is only applied for `fontello`, `icomoon`, `fontawesome5-pro` and `fontawesome6-pro`.
As the user needs to provide their own font files. As the fonts are either generated by those platforms or are proprietary.

This script is actually the most fragile part as if you look at the logic we have to guess where to copy the fonts to.

I'm super open to other ideas on how to approach this.